src/core/agent/services/utils.ts

```diff
@@ -47,7 +47,11 @@ export const OnlineOnly = (
       const errorStack = (error as Error).stack;
       /** If the error is failed to fetch with signify,
        * we retry until the connection is secured*/
-      if (errorStack && /SignifyClient/gi.test(errorStack)) {
+      if (
+        errorStack &&
+        /Failed to fetch/gi.test(errorStack) &&
+        /SignifyClient/gi.test(errorStack)
```

Comment:
imo I would just leave this and `AppWrapper` changes as they were before in case the specific error text changes too much

Reply:
nevermind, of course we need this now to differentiate. I guess we will make sure our e2e tests catch this